.buildkite/scripts/steps/ess_start.sh

```diff
@@ -12,10 +12,19 @@ ess_up $OVERRIDE_STACK_VERSION
 preinstall_fleet_packages
 
 echo "ES_HOST: ${ELASTICSEARCH_HOST}"
-buildkite-agent meta-data set "es.host" $ELASTICSEARCH_HOST
-buildkite-agent meta-data set "es.username" $ELASTICSEARCH_USERNAME
-buildkite-agent meta-data set "es.pwd" $ELASTICSEARCH_PASSWORD
-buildkite-agent meta-data set "kibana.host" $KIBANA_HOST
-buildkite-agent meta-data set "kibana.username" $KIBANA_USERNAME
-buildkite-agent meta-data set "kibana.pwd" $KIBANA_PASSWORD
-buildkite-agent meta-data set "integrations_server.host" $INTEGRATIONS_SERVER_HOST
+echo "BUILDKITE_JOB_ID: ${BUILDKITE_JOB_ID}"
+
+if [[ "${FIPS:-false}" == "true" ]]; then
+    buildkite-agent meta-data set "ess.job.fips" ${BUILDKITE_JOB_ID}
+    echo "FIPS metadata has been set"
+else
+    buildkite-agent meta-data set "ess.job" ${BUILDKITE_JOB_ID}
+fi
+
+buildkite-agent meta-data set "es.host" $ELASTICSEARCH_HOST --job ${BUILDKITE_JOB_ID}
+buildkite-agent meta-data set "es.username" $ELASTICSEARCH_USERNAME --job ${BUILDKITE_JOB_ID}
+buildkite-agent meta-data set "es.pwd" $ELASTICSEARCH_PASSWORD --job ${BUILDKITE_JOB_ID}
+buildkite-agent meta-data set "kibana.host" $KIBANA_HOST --job ${BUILDKITE_JOB_ID}
+buildkite-agent meta-data set "kibana.username" $KIBANA_USERNAME --job ${BUILDKITE_JOB_ID}
+buildkite-agent meta-data set "kibana.pwd" $KIBANA_PASSWORD --job ${BUILDKITE_JOB_ID}
+buildkite-agent meta-data set "integrations_server.host" $INTEGRATIONS_SERVER_HOST --job ${BUILDKITE_JOB_ID}
```

Comment:
let's take an alternative route here, that is gonna be helpful I think 🙂 

```suggestion
METADATA_PREFIX=""
if [[ "${FIPS:-false}" == "true" ]]; then
  METADATA_PREFIX="fips."
  echo "Using FIPS metadata prefix: ${METADATA_PREFIX}"
fi

buildkite-agent meta-data set "${METADATA_PREFIX}es.host" $ELASTICSEARCH_HOST
buildkite-agent meta-data set "${METADATA_PREFIX}es.username" $ELASTICSEARCH_USERNAME
buildkite-agent meta-data set "${METADATA_PREFIX}es.pwd" $ELASTICSEARCH_PASSWORD
buildkite-agent meta-data set "${METADATA_PREFIX}kibana.host" $KIBANA_HOST
buildkite-agent meta-data set "${METADATA_PREFIX}kibana.username" $KIBANA_USERNAME
buildkite-agent meta-data set "${METADATA_PREFIX}kibana.pwd" $KIBANA_PASSWORD
buildkite-agent meta-data set "${METADATA_PREFIX}integrations_server.host" $INTEGRATIONS_SERVER_HOST
```